.drone.yml

```diff
@@ -33,3 +33,7 @@ matrix:
   IMAGE_PHP:
     - fpfis/httpd-php-dev:5.6
     - fpfis/httpd-php-dev:7.1
+
+  COMPOSER_BOUNDARY:
```

Comment:
The matrix is extended with choices for the composer boundary, but this `${COMPOSER_BOUNDARY}` variable is not used in the pipeline, so it has no effect.

For an example on how to use this, see the pipeline of the Code Review component: https://github.com/openeuropa/code-review/blob/master/.drone.yml#L12